### ISSUE TITLE
引入库后报 Type 'FlutterWindow' not found. 

### ISSUE THREAD

**Issue by AndromedaX7** [OpenFlutter/flutter_screenutil#450]
```
../../.pub-cache/hosted/pub.dev/flutter_screenutil-5.6.0/lib/src/screen_util.dart:58:5: Error: Type 'FlutterWindow' not found.
    FlutterWindow? window,
    ^^^^^^^^^^^^^
```
version:5.6.0

fock一份修复试试

**Comment by AndromedaX7** [comment]
• Flutter version 3.7.0-23.0.pre.22 on channel master at /home/juzi/flutter
    • Upstream repository https://github.com/flutter/flutter
    • Framework revision 0bbb5ec0c2 (8 小时前), 2023-01-19 01:22:08 -0500
    • Engine revision 40f7f0f099
    • Dart version 3.0.0 (build 3.0.0-136.0.dev)
    • DevTools version 2.20.0

**Comment by AndromedaX7** [comment]
我已经尝试修复了这个问题，我怎么提交上来

**Comment by gbolahan507** [comment]
whats the soliution pls...am having that issue

**Comment by gbolahan507** [comment]
@AndromedaX7 what is the solution

**Comment by AndromedaX7** [comment]
using stable branch

**Comment by AndromedaX7** [comment]
using stable branch

**Comment by AndromedaX7** [comment]
Or modify library source code.
1. lib/src/screen_util.dart
2. lib/src/screenutil_init.dart
https://github.com/OpenFlutter/flutter_screenutil/pull/451/files

**Comment by gbolahan507** [comment]
@AndromedaX7 Thanks

**Comment by lizhuoyuan** [comment]
fixed at  5.6.1
